src/providers/PortfolioProvider/hooks/usePositionsData.ts

```
@@ -0,0 +1,203 @@
+    const dates = successfulQueries.map((query) => {
+      const metaUpdatedAt = query.data?.meta?.updatedAt;
+      return metaUpdatedAt
+        ? new Date(metaUpdatedAt)
+        : new Date(query.dataUpdatedAt);
+    });
+
+    return dates.length > 0 ? min(dates).getTime() : null;
+  }, [successfulQueries]);
+
+  const refetch = useCallback(() => {
+    queries.forEach((query) => query.refetch());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)